src/main/java/com/world/entertainment/worldentertainment/entity/Country.java

```diff
@@ -0,0 +1,87 @@
+package com.world.entertainment.worldentertainment.entity;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.Set;
+
+@Entity
+@Table(schema = "public")
+public class Country {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @OneToMany(mappedBy = "country")
+    private Set<Entertainment> entertainments;
+
+    private String name;
+    private String code;
+
+    private Date createAt;
+    private Date updateAt;
+    private Date deleteAt;
+
+    public int getId() {
+        return id;
+    }
+
+    public Country setId(int id) {
+        this.id = id;
+        return this;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Country setName(String name) {
+        this.name = name;
+        return this;
+    }
+
+    public String getCode() {
+        return code;
+    }
+
+    public Country setCode(String code) {
+        this.code = code;
+        return this;
+    }
+
+    public Set<Entertainment> getEntertainments() {
```

Comment:
I guess Country shod contains cities and cities shod contains Entertainments. Let's add that relation

Reply:
changed hierarchy